### DISCUSSION TITLE
Better shortcuts

### DISCUSSION THREAD

**Discussion by Greytsu** [dailydotdev/daily#504] [Feature Requests]
Hey I have been using daily.dev for a few days and the only thing that bothers me is the shortcuts.

I think that centering the shortcuts with bigger icons would be way more efficient.

Thank you 😄

**Comment by idoshamun** [comment]
Hi @Greytsu,

Thank you for the suggestion.
We don't have plans to change the shortcuts in the near future.
We prefer to reserve more space for the cards so we can show more articles.
